packages/app/src/cli/commands/app/function/replay.ts

```diff
@@ -0,0 +1,83 @@
+import {functionFlags, inFunctionContext} from '../../../services/function/common.js'
+import {replay} from '../../../services/function/replay.js'
+import {appFlags} from '../../../flags.js'
+import {showApiKeyDeprecationWarning} from '../../../prompts/deprecation-warnings.js'
+import {environmentVariableNames} from '../../../constants.js'
+import Command from '@shopify/cli-kit/node/base-command'
+import {globalFlags} from '@shopify/cli-kit/node/cli'
+import {Flags} from '@oclif/core'
+import {getEnvironmentVariables} from '@shopify/cli-kit/node/environment'
+import {isTruthy} from '@shopify/cli-kit/node/context/utilities'
+
+export default class FunctionReplay extends Command {
+  static hidden = true
+  static summary = 'Replays a function run from an app log.'
+
+  static descriptionWithMarkdown = `Runs the function from your current directory for [testing purposes](https://shopify.dev/docs/apps/functions/testing-and-debugging). To learn how you can monitor and debug functions when errors occur, refer to [Shopify Functions error handling](https://shopify.dev/docs/api/functions/errors).`
+
+  static description = this.descriptionWithoutMarkdown()
+
+  static flags = {
+    ...globalFlags,
+    ...appFlags,
+    ...functionFlags,
+    'api-key': Flags.string({
+      hidden: true,
+      description: "Application's API key",
+      env: 'SHOPIFY_FLAG_API_KEY',
+      exclusive: ['config'],
+    }),
+    'client-id': Flags.string({
+      hidden: false,
+      description: "Application's Client ID",
+      env: 'SHOPIFY_FLAG_CLIENT_ID',
+      exclusive: ['config'],
+    }),
+    export: Flags.string({
+      char: 'e',
+      hidden: false,
+      description: 'Name of the Web Assembly export to invoke.',
+      default: '_start',
+      env: 'SHOPIFY_FLAG_EXPORT',
+    }),
```

Comment:
💭 Isn't the export being invoked defined by the run itself? E.G. a run using `fetch` couldn't be sensibly rerun using `run`. Perhaps that should be part of the payload so you never have to specify it here?

Probably not urgent for this PR, but would be an improvement for later.

Reply:
🤔 It's one of the flags that we can invoke function runner with so that's how it ended up here. 
The export is defined in the extension's `shopify.extension.toml`, so I think we could maybe improve this by just reading from there?

Reply:
I agree with @DuncanUszkay1 that this should be part of the payload, the input you are replaying would be specific to a certain target.

Reply:
Will add that to the payload in my upcoming PR that also adds source/source namespace